cloudformation.yaml

```diff
@@ -225,15 +259,32 @@ Resources:
                  "Next": "MigrateShards",
                  "Retry": [ {
                    "ErrorEquals": [ "States.ALL" ],
-                   "IntervalSeconds": 10,
-                   "MaxAttempts": 12
+                   "IntervalSeconds": 30,
+                   "MaxAttempts": 20,
+                   "BackoffRate": 1.0
                   } ]
                },
                "MigrateShards": {
                   "Type": "Task",
                   "Resource": "${MigrateShardsArn}",
-                  "End": true
-               }
+                  "Next": "ShardMigrationCheck"
```

Comment:
We could consider adding a [Wait state](https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-wait-state.html) here, rather than polling continuously - presumably this migration will take some time, so maybe there's no point polling immediately?